packages/backend/server/src/core/auth/controller.ts

```diff
@@ -147,6 +152,33 @@ export class AuthController {
       if (!allowSignup) {
         throw new SignUpForbidden();
       }
+
+      const requireEmailDomainVerification = await this.config.runtime.fetch(
```

Comment:
Will this cause registered users to be unable to log into their accounts?
We should only check this when registering

Reply:
this logic only execute that user not register yet